test/test_ops.py

```diff
@@ -69,7 +69,8 @@ def tearDownClass(cls):
     def test_dtypes(self, device, op):
         # Check complex32 support only if the op claims.
         # TODO: Once the complex32 support is better, we should add check for complex32 unconditionally.
-        include_complex32 = ((torch.complex32,) if op.supports_dtype(torch.complex32, device) else ())
+        device_type = torch.device(device).type
```

Comment:
device for CUDA is `cuda:0` with index.

It worked for all the operators previously added because they don't specify different dtypes for CUDA.

Implementation reference:
https://github.com/pytorch/pytorch/blob/be354d8139f70c5fabfce882671ad1aed057b1ba/torch/testing/_internal/common_methods_invocations.py#L933-L939